### DISCUSSION TITLE
FASTA digest turns itself ON when starting search - unwantedly

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#497] [Q&A]
I noticed that a starting a library-based search sometimes triggers that the box for 'FASTA digest for library-free search / library generation' turns itself on. The tick does not appear immediately but when clicking to another step in the pipeline mode and then returning to the step that has started the tick is suddenly there. 
Q1: Why?
Q2: Does it affect the search results ? (If I use a library and have disabled protein inference)

**Comment by vdemichev** [comment]
This is weird. This option absolutely should not be used together with a spectral library, when FASTA is also specified. Need to 'untick' this then... Without FASTA will have no effect.
